### PR TITLE
Bump package versions to work with recent react-sketchapp

### PR DESCRIPTION
**Pull request by lessthanzero**
Tested with react-sketchapp 2.1.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +27,12 @@
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-no-not-accumulator-reassign": "^0.1.0",
     "eslint-plugin-prettier": "^2.3.1",
+    "eslint-plugin-react": "^7.4.0",
     "lint-staged": "^4.2.3",
     "pre-commit": "^1.2.2",
     "prettier": "^1.7.4",
-    "skpm-builder": "^0.1.1"
+    "skpm-builder": "^0.1.1",
+    "webpack": "^2.2.0"
```
Comment (mathieudutour) [inline package.json]: why do we need webpack here?

Comment (lessthanzero) [reply package.json]: I've kept getting warnings during `npm i` stage telling me that webpack is an optional dependency that must be installed manually. Maybe it's not important at all

Comment (mathieudutour) [reply package.json]: hum it's probably better fixed by removing `skpm-builder` and using `@skpm/builder` instead (`skpm-builder` is like super old)

Comment (lessthanzero) [reply package.json]: @mathieudutour can I do it by a follow-up commit?)

Comment (mathieudutour) [reply package.json]: commit or PR?

Actually, let me just add you as a collaborator so you have write access. Feel free to ping me on PR where you want reviews of course

**Comment by lessthanzero** [comment]
@mathieudutour back trying to tackle the dependencies. Now, when I'm trying to use @skpm/builder package, `npm i` keeps pointing out errors in `type` imports in sketch-library.js and some other files. I don't think I understand the internal workings good enough to solve the problem, and I need some help.

**Comment by mathieudutour** [comment]
What are the errors?

**Comment by lessthanzero** [comment]
Here's full output with `@skpm/builder` instead of `skpm-builder` + `webpack` in package.json:

```
➜  react-sketchapp-library git:(master) npm i

> react-sketch-library@0.1.0 postinstall /Users/agkatin/Developer/react-sketchapp-library
> npm run build && skpm-link


> react-sketch-library@0.1.0 build /Users/agkatin/Developer/react-sketchapp-library
> skpm-build

[1/2] 🖨  Copied src/manifest.json in 23ms
error Error while building ./sketch-library.js
./src/sketch-library.js
Module build failed (from ./node_modules/@skpm/builder/node_modules/babel-loader/lib/index.js):
SyntaxError: /Users/agkatin/Developer/react-sketchapp-library/src/sketch-library.js: Unexpected token (7:12)

   5 | import { render, TextStyles, View } from 'react-sketchapp';
   6 | import designSystem from './designSystem';
>  7 | import type { DesignSystem } from './designSystem';
     |             ^
   8 |
   9 | import exposeSymbols from './exposeSymbols'
  10 |
    at _class.raise (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:3939:15)
    at _class.unexpected (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:5248:16)
    at _class.expectContextual (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:5216:41)
    at _class.parseImport (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:8418:12)
    at _class.parseStatementContent (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:7226:27)
    at _class.parseStatement (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:7145:17)
    at _class.parseBlockOrModuleBlockBody (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:7696:23)
    at _class.parseBlockBody (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:7683:10)
    at _class.parseTopLevel (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:7110:10)
    at _class.parse (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:8510:17)
    at parse (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/parser/lib/index.js:10462:38)
    at parser (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/core/lib/transformation/normalize-file.js:170:34)
    at normalizeFile (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/core/lib/transformation/normalize-file.js:138:11)
    at runSync (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/core/lib/transformation/index.js:44:43)
    at runAsync (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/core/lib/transformation/index.js:35:14)
    at process.nextTick (/Users/agkatin/Developer/react-sketchapp-library/node_modules/@babel/core/lib/transform.js:34:34)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
    at process._tickCallback (internal/process/next_tick.js:181:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-sketch-library@0.1.0 build: `skpm-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-sketch-library@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/agkatin/.npm/_logs/2018-09-26T10_55_42_968Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-sketch-library@0.1.0 postinstall: `npm run build && skpm-link`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-sketch-library@0.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/agkatin/.npm/_logs/2018-09-26T10_55_43_018Z-debug.log
```

**Comment by lessthanzero** [comment]
@mathieudutour done some minor version updates (though it hasn't eliminated warnings completely, but I guess it's not important, after all) and experimentally learned that `@skpm/builder v0.5.0` introduces some breaking changes leading to build errors like the one above. So for the time being I'd stick with `@skpm/builder v0.4.3`

Tentatively merging into master.

**Pull request merged**